src/main/java/com/dev/cinema/models/Movie.java

```diff
@@ -0,0 +1,47 @@
+package com.dev.cinema.models;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@Entity
+@Table(name = "movies")
+public class Movie {
+    @Id
+    @Column(name = "movie_id")
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+        private Long id;
+    private String title;
+    private String description;
+
+    public Long getId() {
+        return id;
+    }
+
+    public void setId(Long id) {
+        this.id = id;
+    }
+
+    public String getTitle() {
+        return title;
+    }
+
+    public void setTitle(String title) {
+        this.title = title;
+    }
+
+    public String getDescription() {
+        return description;
+    }
+
+    public void setDescription(String description) {
+        this.description = description;
+    }
```

Comment:
what do you see in your console when you run the main class?

Reply:
was description=null
but i`ll change to "No description provided"

Reply:
I see that you use Lombok
no changes are needed, sorry